featurebyte/models/feature.py

```diff
@@ -68,6 +60,20 @@ class TableIdColumnNames(FeatureByteBaseModel):
     column_names: List[str]
 
 
+class OfflineStoreIngestQueryGraph(FeatureByteBaseModel):
+    """
+    OfflineStoreIngestQuery object stores the offline store ingest query for a feature or target.
```

Comment:
Is it possible to include primary entity that the ingest query graph corresponds to? I think this information is likely needed by the consumer of OfflineStoreIngestQueryGraph.

Reply:
Yes, we can include that information. Will update this to include primary entity ids.

Reply:
Updated to include primary entity ids in this model.